The suggested change does not work, and produces the error:

```bash
/home/desaiin/dune-modules/dumux-adapter/examples/macro-micro/micro-heat/micro_sim.cpp:268:48: error: ‘__deepcopy__’ is not a member of ‘MicroSimulation’
  268 |         .def("__deepcopy__", &MicroSimulation::__deepcopy__);
      |                                                ^~~~~~~~~~~~
```
@erikscheurer any idea what is going wrong? Lets use this repository to test whatever we are changing in the Micro Manager.

--- a/micro-heat/micro_sim.cpp
+++ b/micro-heat/micro_sim.cpp
@@ -66,6 +66,7 @@ public:
 
     void save_checkpoint();
     void reload_checkpoint();
+    MicroSimulation __deepcopy__(py::dict memo);
 
 private:
     const double pi_ = 3.14159265358979323846;

